addons/website_event_track_quiz/models/event_quiz.py

```diff
@@ -59,13 +67,8 @@ class QuizAnswer(models.Model):
     sequence = fields.Integer("Sequence")
     question_id = fields.Many2one('event.quiz.question', string="Question", required=True, ondelete='cascade')
     text_value = fields.Char("Answer", required=True, translate=True)
-    is_correct = fields.Boolean("Is correct answer", compute='_compute_is_correct')
-    comment = fields.Text("Comment", translate=True,
+    is_correct = fields.Boolean("Correct", default=False)
```

Comment:
Original version allowed to have several "correct" answers, depending on points. Why do we need to move to a stricter implementation ? It is still a light quiz model with points that have no effects, not sure we should be stricter than necessary ...